core/src/_story.ts

```diff
@@ -1,7 +1,12 @@
 const arg = (target: object | null | string) => {
-	if (target !== null && typeof target === "object") {
-		return { control: { type: "radio", options: Object.keys(target) } }
-	} else {
+	if (target === null) {
+		return
```

Comment:
Are you sure about this? I think it should be `{ control: { type: null } }` (as in the previous code). However may be that's the same thing with `undefined`?